partiql-playground/package-lock.json

```diff
@@ -0,0 +1,1069 @@
+{
```

Comment:
This is an auto-generated file, but is [required](https://nodejs.dev/en/learn/the-package-lock-json-file) to get check-in—you can ignore reviewing it:

>The package-lock.json file needs to be committed to your Git repository, so it can be fetched by other people, if the project is public or you have collaborators, or if you use Git as a source for deployments.